src/pilot/deploy-overcloud.py

```diff
@@ -430,19 +997,27 @@ def main():
         start = time.time()
         status = run_deploy_command(cmd)
         end = time.time()
-        print '\nExecution time: {} (hh:mm:ss)'.format(
-            time.strftime('%H:%M:%S', time.gmtime(end - start)))
+        logger.info('\nExecution time: {} (hh:mm:ss)'.format(
+            time.strftime('%H:%M:%S', time.gmtime(end - start))))
         print 'Fetching SSH keys...'
         update_ssh_config()
         if status == 0:
             horizon_url = finalize_overcloud()
+            print("\nDeployment Completed")
+            create_aggregates(args.enable_hugepage, args.enable_numa,
```

Comment:
It seems like this chunk of code (create_aggregates, create_custom_flavors, and update_filters should be moved into finalize_overcloud() since that method is responsible for doing these types of things.

Reply:
Agree. Moved.